dev-python/matplotlib-venn/matplotlib-venn-0.11.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Routines for plotting area-weighted two- and three-circle venn diagrams"
+HOMEPAGE="http://pysurfer.github.com"
+SRC_URI="'mirror://pypi/m/${PN}/${P}.zip"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+RDEPEND="
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]"
+DEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
+
+python_test() {
+	nosetests -v || die
```

Comment:
so I should put this inside the test phase:

```
    - cd ${SRC_DIR}
    - nosetests --with-timer --timer-top-n 10
    - make flake
``` 

instead of `nosetests -v || die` ?